app-backend/db/src/main/scala/CampaignDao.scala

```diff
@@ -0,0 +1,118 @@
+package com.rasterfoundry.database
```

Comment:
x-ref -- if merged before #5393, this file will need the new Postgres time imports from more recent doobie